client/components/tinymce/plugins/insert-menu/style.scss

```diff
@@ -0,0 +1,31 @@
+.mce-container .mce-wpcom-insert-menu {
+	& .mce-caret {
+		padding-left: 12px;
+	}
+}
+
+.mce-container.mce-menu .mce-container-body .mce-wpcom-insert-menu__menu-item {
+	margin-left: -6px;
+
+	& svg {
```

Comment:
Thanks @mtias and @aduth. This is still very much a work-in-progress. I have rearranged the CSS to make better use of classes for specifying the styles, but I'm still pretty troubled by the strange `.mce` classes.

I'm new to TinyMCE work and had trouble getting enough specificity to overwrite the TinyMCE-provided styles. Any help to clean this up would be much-appreciated!